test/e2e/performanceprofile/functests/8_performance_workloadhints/workloadhints.go

```diff
@@ -258,28 +259,32 @@ var _ = Describe("[rfe_id:49062][workloadHints] Telco friendly workload specific
 
 		When("realtime and high power consumption enabled", func() {
 			It("[test_id:50993][crit:high][vendor:cnf-qe@redhat.com][level:acceptance]should update kernel arguments and tuned accordingly", func() {
+				currentWorkloadHints := profile.Spec.WorkloadHints
```

Comment:
do we need a DeepCopy() here?

Reply:
You mean should we DeepCopy() ?

Reply:
Here, i only intend to get the current profile before i modify, to check if the current profile is already contains the changes i intend to make further, if the current profile already contains the changes needed,  then don't need to apply the spec changes , directly check the tuning parameters.